databricks_cli/pipelines/cli.py

```diff
@@ -229,7 +264,7 @@ def run_cli(api_client, pipeline_id):
 @provide_api_client
 def stop_cli(api_client, pipeline_id):
     """
-    Stops the execution of a delta pipelines run by terminating the cluster. Processing of data can
+    Stops the pipeline run by terminating the cluster. Processing of data can
```

Comment:
This is not accurate any longer. I think it should say:
"Stops the pipelines by cancelling any active run".
and nothing about resuming processing, because that is only true in a few cases

Reply:
done